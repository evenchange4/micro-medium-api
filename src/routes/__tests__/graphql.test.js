const graphql = require('../graphql');

jest.mock('graphql-server-micro', () => ({
  microGraphql: d => d,
}));

jest.mock('../../graphql/schema', () => ({ schema: 'mock' }));

it('should return graphql arguments', () => {
  expect(graphql()).toMatchSnapshot();
});
