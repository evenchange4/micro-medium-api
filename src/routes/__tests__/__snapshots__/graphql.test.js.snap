// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return graphql arguments 1`] = `
Object {
  "context": Object {
    "Raw": DataLoader {
      "_batchLoadFn": [Function],
      "_options": Object {
        "cacheKeyFn": [Function],
      },
      "_promiseCache": Map {},
      "_queue": Array [],
    },
  },
  "schema": Object {
    "schema": "mock",
  },
}
`;
